less/home-page.less

```diff
@@ -45,9 +67,23 @@
     padding: 50px 60px;
     display: flex;
     justify-content: center;
+    @media @phone {
+      padding: 0px;
+      display: flex;
```

Comment:
Same thing here: you don't need to reassign display: flex to this component in the media query as it's already on the base